lib/bundler/fetcher/index.rb

```diff
@@ -30,7 +30,8 @@ def fetch_spec(spec)
 
         uri = URI.parse("#{remote_uri}#{Gem::MARSHAL_SPEC_DIR}#{spec_file_name}.rz")
         if uri.scheme == "file"
-          Bundler.load_marshal Bundler.rubygems.inflate(Gem.read_binary(uri.path))
+          path = Gem.win_platform? ? uri.path[1..-1] : uri.path
```

Comment:
duplicated code, is it possible to avoid these duplication?

Reply:
Maybe, I'm not sure why this code is currently duplicated, but refactoring that is out of the scope of this PR.

Reply:
I think this comment referred specifically to the code `path = Gem.win_platform? ? uri.path[1..-1] : uri.path` you added - but as it seems the whole `fetch_spec` is duplicated this is indeed out of scope to extract this, considering that `fetch_spec` itself will probably be dedpulicated sooner or later (hopefully).

Reply:
Correct :+1: